.github/workflows/main.yaml

```diff
@@ -0,0 +1,29 @@
+name: "Main"
+
+on: # yamllint disable-line rule:truthy
+  pull_request: null
+  push:
+    branches:
+      - "main"
+
+jobs:
+  monorepo-validate:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v4"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "8.2"
```

Comment:
Should we use PHP 8.1, as this version is the minimum requirement of TYPO3? Or is this the minimum version of the parser?

Reply:
We can use php 8.1 here